src/node/nodestate.h

```diff
@@ -233,35 +314,9 @@ namespace ccf
       {
         case StartType::New:
         {
-          GenesisGenerator g(network);
-          g.init_values();
-
-          for (auto& cert : args.config.genesis.member_certs)
-            g.add_member(cert);
-
-          // Add self as TRUSTED
-          self = g.add_node({args.config.node_info_network,
-                             node_cert,
-                             quote,
-                             NodeStatus::TRUSTED});
-
-#ifdef GET_QUOTE
-          // Trust own code id
-          g.trust_code_id(node_code_id);
-#endif
-
-          // set access whitelists
-          // TODO(#feature): this should be configurable
-          for (const auto& wl : default_whitelists)
-            g.set_whitelist(wl.first, wl.second);
-
-          g.set_gov_scripts(lua::Interpreter().invoke<nlohmann::json>(
-            args.config.genesis.gov_script));
-
           network.secrets = std::make_unique<NetworkSecrets>(
             "CN=The CA", std::make_unique<Seal>(writer_factory));
-
-          g.create_service(network.secrets->get_current().cert);
+          self = 0;
```

Comment:
Minor: Could you add a comment for this, saying that the first node's id is always 0.